google/resource_compute_network.go

```diff
@@ -195,9 +195,12 @@ func resourceComputeNetworkDelete(d *schema.ResourceData, meta interface{}) erro
 		return err
 	}
 
-	// Delete the network
+	return deleteComputeNetwork(project, d.Id(), config)
```

Comment:
You'll still want a d.SetId("") if err == nil, right?

Reply:
The `d.SetId("")` is actually redundant: https://github.com/hashicorp/terraform/blob/master/helper/schema/resource.go#L205

Reply:
I did not know that!